### PR TITLE
Adding sms provider SMSGlobal

### PR DESCRIPTION
**Pull request by bosdla**
Hi guys,
I've added SMSGlobal.com as 2fa SMS Provider. (similar to playSMS) 

Can you please integrate this into the main version?
thanks you
Bosdla

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@seyfahni @j-ed @boppy @fblackburn1 @kimsyversen could you help review this? :)

File: lib/Service/Gateway/SMS/Provider/SMSGlobalConfig.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
```
Comment (ChristophWurst) [inline lib/Service/Gateway/SMS/Provider/SMSGlobalConfig.php:6]: it wasn't me :-)

File: lib/Service/Gateway/SMS/Provider/SMSGlobal.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @author Pascal Clémot <pascal.clemot@free.fr>
```
Comment (ChristophWurst) [inline lib/Service/Gateway/SMS/Provider/SMSGlobal.php:6]: I don't think so

File: doc/Admin Documentation.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ Interactive admin configuration:
 occ twofactorauth:gateway:configure sms
 ```
 
+### SMSGlobal 
+URL: https://www.smsglobal.com/
+Stability: Experimental
+
+Use the Webservices provided by playSMS for sending SMS.
```
Comment (boppy) [inline doc/Admin Documentation.md:22]: Please Change name here.

File: lib/Service/Gateway/SMS/Provider/SMSGlobal.php [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+	/**
+	 * @param string $identifier
+	 * @param string $message
+	 *
+	 * @throws SmsTransmissionException
+	 */
+	public function send(string $identifier, string $message) {
+		$config = $this->getConfig();
+                $to=str_replace("+", "", $identifier);
```
Comment (boppy) [inline lib/Service/Gateway/SMS/Provider/SMSGlobal.php:54]: Please keep `\t` indentation.

File: lib/Command/Configure.php [hunk 5]
```diff
@@ -334,6 +335,22 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setUser($username);
 				$providerConfig->setApiKey($apiKey);
 
+				break;
+			case 'smsglobal':
+				$config->setProvider($provider);
+				/** @var SMSGlobalConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+
+				$url='https://api.smsglobal.com/http-api.php';
```
Comment (boppy) [inline lib/Command/Configure.php:344]: Since this is not meant to be configured, you could hard-code it...

**Review by boppy — CHANGES_REQUESTED** [review]

File: lib/Service/Gateway/SMS/Provider/SMSGlobal.php (hunk 4 above)
Comment (boppy) [reply lib/Service/Gateway/SMS/Provider/SMSGlobal.php]: Still not a `\t` after fix...

File: lib/Service/Gateway/SMS/Provider/SMSGlobal.php [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+	/**
+	 * @param string $identifier
+	 * @param string $message
+	 *
+	 * @throws SmsTransmissionException
+	 */
+	public function send(string $identifier, string $message) {
+		$config = $this->getConfig();
+        $to=str_replace("+", "", $identifier);
```
Comment (boppy) [inline lib/Service/Gateway/SMS/Provider/SMSGlobal.php:54]: Still not a `\t` after fix...

**Review by boppy — CHANGES_REQUESTED** [review]

**Pull request merged**